src/datasources/organisations/entities/organisations.entity.db.ts

```diff
@@ -0,0 +1,32 @@
+import { Column, Entity, Index, PrimaryGeneratedColumn } from 'typeorm';
+import {
+  OrganisationStatus,
+  Organisation as DomainOrganisation,
+} from '@/domain/organisations/entities/organisation.entity';
+
+@Entity('organisations')
+export class Organisations implements DomainOrganisation {
+  @PrimaryGeneratedColumn({ primaryKeyConstraintName: 'PK_org_id' })
+  id!: number;
+
+  @Column({ type: 'varchar', length: 255 })
+  name!: string;
+
+  @Index('idx_org_status')
+  @Column({
+    type: 'integer',
+  })
+  status!: OrganisationStatus;
```

Comment:
This is out of the scope of this PR, but a thought I had re. numeric enums. What do you think about leveraging the `transform` method to convert this to <> from a string?

Reply:
Yes, agree handling strings rather than integers would make the code easier to read while still having the benefit of storing and indexing it more efficiently. @PooyaRaki do you have an opinion on using transformers for the enums we create?

Reply:
As we're using an enum, it should be fine as is for now but it's something we can consider in the future.